templates/cluster-template-talos-v0.13.4.yaml

```diff
@@ -38,7 +38,7 @@ spec:
   template:
     spec:
       serverClassRef:
-        apiVersion: metal.sidero.dev/v1alpha1
+        apiVersion: metal.sidero.dev/v1alpha2
```

Comment:
should be bump to v1alpha2 for the talos v0.13.4 template?

Reply:
yep, as Sidero still supports 0.13 (we are testing with it), so we should keep the template up to date with the resources